cf/i18n/resources/en_US.all.json

```diff
@@ -3645,6 +3645,16 @@
       "modified": false
    },
    {
+      "id": "Optional context path. Path must include at least one character following a leading forward slash (/). Trailing slashes will be stripped. Requests received with a trailing slash will match.",
```

Comment:
@crhino can you explain this a little more? If trailing slashes are being removed, then what does " Requests received with a trailing slash will match." mean? Do you mean to say that they'll be matched after the trailing slashed is removed?